transaction/src/model/instruction.rs

```diff
@@ -95,15 +82,15 @@ pub enum Instruction {
     ///
     /// Buckets and proofs in arguments moves from transaction context to the callee.
     CallFunction {
-        fn_identifier: FnIdentifier,
+        fn_identifier: FunctionIdent,
```

Comment:
Minor: Should have consistent name like `function_identifier`

Reply:
good point